contentcuration/contentcuration/settings.py

```diff
@@ -354,14 +354,9 @@ def ugettext(s):
 CELERY_ACCEPT_CONTENT = ['application/json']
 CELERY_TASK_SERIALIZER = 'json'
 CELERY_RESULT_SERIALIZER = 'json'
-# This is needed for worker update_state calls to work so they can send progress info.
-CELERYD_STATE_DB = '/tmp/celery_state'
 # If this is True, Celery tasks are run synchronously. This is set to True in the unit tests,
 # as it is not possible to correctly test Celery tasks asynchronously currently.
 CELERY_TASK_ALWAYS_EAGER = False
-# This tells Celery to mark a task as started. Otherwise, we would have no way of tracking
-# if the task is running.
-CELERY_TASK_TRACK_STARTED = True
```

Comment:
Moved this to the `CeleryTask` class, so it's closer to the task definition. We likely don't want to change this value anyway.